vsftpd/vsftpd.spec

```diff
@@ -0,0 +1,157 @@
+###############################################################################
+
+%define _posixroot        /
+%define _root             /root
+%define _bin              /bin
+%define _sbin             /sbin
+%define _srv              /srv
+%define _home             /home
+%define _lib32            %{_posixroot}lib
+%define _lib64            %{_posixroot}lib64
+%define _libdir32         %{_prefix}%{_lib32}
+%define _libdir64         %{_prefix}%{_lib64}
+%define _logdir           %{_localstatedir}/log
+%define _rundir           %{_localstatedir}/run
+%define _lockdir          %{_localstatedir}/lock/subsys
+%define _cachedir         %{_localstatedir}/cache
+%define _spooldir         %{_localstatedir}/spool
+%define _crondir          %{_sysconfdir}/cron.d
+%define _loc_prefix       %{_prefix}/local
+%define _loc_exec_prefix  %{_loc_prefix}
+%define _loc_bindir       %{_loc_exec_prefix}/bin
+%define _loc_libdir       %{_loc_exec_prefix}/%{_lib}
+%define _loc_libdir32     %{_loc_exec_prefix}/%{_lib32}
+%define _loc_libdir64     %{_loc_exec_prefix}/%{_lib64}
+%define _loc_libexecdir   %{_loc_exec_prefix}/libexec
+%define _loc_sbindir      %{_loc_exec_prefix}/sbin
+%define _loc_bindir       %{_loc_exec_prefix}/bin
+%define _loc_datarootdir  %{_loc_prefix}/share
+%define _loc_includedir   %{_loc_prefix}/include
+%define _rpmstatedir      %{_sharedstatedir}/rpm-state
+%define _pkgconfigdir     %{_libdir}/pkgconfig
+
+%define __service         %{_sbin}/service
+%define __chkconfig       %{_sbin}/chkconfig
+%define __useradd         %{_sbindir}/useradd
+%define __groupadd        %{_sbindir}/groupadd
+%define __getent          %{_bindir}/getent
+
+###############################################################################
+
+%define service_user         vsftpd
+%define service_group        vsftpd
+%define service_name         %{name}
+%define service_home         %{_localstatedir}/ftp
+
+###############################################################################
+
+Summary:              Very Secure FTP Daemon
+Name:                 vsftpd
+Version:              3.0.3
+Release:              0%{?dist}
+License:              GPL
+Group:                System Environment/Daemons
+URL:                  http://vsftpd.beasts.org/
+
+Source0:              https://security.appspot.com/downloads/%{name}-%{version}.tar.gz
+Source1:              %{name}.init
+Source2:              %{name}.conf
+Source3:              %{name}.logrotate
+Source4:              %{name}.sysconfig
+
+BuildRoot:            %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+Requires:             openssl >= 1.0.1e, kaosv >= 2.7.0, logrotate
```

Comment:
Dependency list not require commas, it's optional and not used in other packages.